packages/ERTP/src/payment.js

```diff
@@ -2,19 +2,13 @@
 
 import { initEmpty } from '@agoric/store';
 
-/** @typedef {import('@endo/patterns').MethodGuard} MethodGuard */
-/**
- * @template {Record<string | symbol, MethodGuard>} [T=Record<string | symbol, MethodGuard>]
- * @typedef {import('@endo/patterns').InterfaceGuard<T>} InterfaceGuard
- */
-/** @typedef {import('@agoric/zone').Zone} Zone */
-
+// TODO Type InterfaceGuard better than InterfaceGuard<any>
```

Comment:
Need help. Finally resorted to `any` after everything else I tried failed. I don't understand why this problem arose in this PR but not prior to this PR.

I got rid of all the `@typedef`s in case it could help with this, which it did not. But I left the non-@typedef form in anyway because it is now our recommended practice. Preserves type info better in the IDE.